examples/functions-with-defaultable-parameters/functions_with_defaultable_parameters.bal

```diff
@@ -16,24 +16,36 @@ function printSalaryDetails(int baseSalary,
 }
 
 public function main() {
-    // Calls the function by passing only the `baseSalary`.
-    // The `annualIncrement` and the `bonusRate` defaults to 20 and 0.02
+    // Call the function by passing only the `baseSalary`.
+    // The `annualIncrement` and the `bonusRate` default to 20 and 0.02
     // respectively.
     printSalaryDetails(2500);
 
-    // Calls the function by passing only the `baseSalary` and `annualIncrement`.
-    // Passes the defaultable parameter as a key-value pair when invoking the
-    // function. The `bonusRate` defaults to 0.02.
+    // Call the function by passing only the `baseSalary` and `annualIncrement`.
+    // `annualIncrement` is passed as a named argument.
+    // The `bonusRate` defaults to 0.02.
     printSalaryDetails(2500, annualIncrement = 100);
 
-    // Calls the function by passing only the `baseSalary` and `bonusRate`.
+    // Call the function by passing only `baseSalary` and `annualIncrement`
+    // again, but now pass `annualIncrement` as a positional argument.
+    // The `bonusRate` defaults to 0.02.
+    printSalaryDetails(2500, 100);
+
+    // Call the function by passing only the `baseSalary` and `bonusRate`.
     // The `annualIncrement` defaults to 20.
     printSalaryDetails(2500, bonusRate = 0.1);
 
-    // Calls the function by passing all three parameters.
+    // In order to pass `bonusRate` as a positional argument, a value would
+    // have to be specified for `annualIncrement` too.
+    // All arguments are positional arguments here.
```

Comment:
```suggestion
    // All the below arguments are positional arguments.
```

Reply:
Comments in BBEs are rendered side-by-side with the code, "below" wouldn't work. :)